command/agent/job_endpoint.go

```diff
@@ -719,6 +699,50 @@ func (s *HTTPServer) JobsParseRequest(resp http.ResponseWriter, req *http.Reques
 	return jobStruct, nil
 }
 
+func regionForJob(job *api.Job, queryRegion, agentRegion string) (string, string) {
```

Comment:
:100: 
this stuff should have been a stand-alone method even before MRD came into the picture